.env.example

```diff
@@ -0,0 +1,46 @@
+############
+# REQUIRED #
+############
+UTA_DB_URL=postgresql://anonymous@localhost:5432/uta/uta_20241220			# See `docs/uta.md` for more info
+SEQREPO_DATAPROXY_URI=seqrepo+file:///usr/local/share/seqrepo/2024-12-20	# See `docs/seqrepo` for more info
+ANYVAR_STORAGE_URI=postgresql://anyvar:anyvar-pw@localhost:5432/anyvar		# Set to `null` if running AnyVar without storage
```

Comment:
based on https://github.com/biocommons/anyvar/blob/964e7c274047011e343e2c9d477e518a474fc29b/src/anyvar/anyvar.py#L66

I think this actually needs to be an empty string, not the string "null", to run w/o storage. Needs to be fixed in a few other places too.